must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# Expect base collection path as an argument
+BASE_COLLECTION_PATH=$1
+
+# Use PWD as base path if no argument is passed
+if [ "${BASE_COLLECTION_PATH}" = "" ]; then
+    BASE_COLLECTION_PATH=$(pwd)
+fi
+
+# Make a globle variable for namespace
+install_namespace=openshift-storage
+
+# Add general resources to list if necessary
+
+# Resource List
+resources=()
+# collect storagecluster resources
+
+# TODO: Re enable the collection of storagecluster via inspect command
+# resources+=(storageclusters)
+
+# collect OB/OBC resoureces
+resources+=(objectbucketclaims)
+resources+=(objectbuckets)
+
+# collection path for OC commands
+mkdir -p ${BASE_COLLECTION_PATH}/oc_output/
+
+# Command List
+commands_get=()
+
+# collect oc output of OC get commands
+commands_get+=("pods -owide")
+commands_get+=("nodes --show-labels")
+commands_get+=("pvc --all-namespaces")
```

Comment:
Remove this, you perform this action again at the end of the file, so it's redundant.